### ISSUE TITLE
Program does not allow me to add a custom version folder

### ISSUE THREAD

**Issue by Kewlname** [MultiMC/MultiMC5#1707]
I am not sure if I am missing it or if it does not exist, but I am trying to add a Wurst hack client into the versions folder, but can not find a way to do so. Thank You. From Me.

**Comment by peterix** [comment]
Try to add it as a jar mod on top of the equivalent Minecraft version (in the Version tab of the instance).

**Comment by Kewlname** [comment]
I tried to avoid bugging you again, but I can not seem to figure out how to add it as a jar mod. I tried using the jar directly inside the Wurst client, and tried to use the original .zip file. Can you explain it to me again but dumb it down for me? Thanks.

**Comment by peterix** [comment]
![image](https://cloud.githubusercontent.com/assets/203326/20004484/eccdc68a-a28c-11e6-96d1-0c5ee4091443.png)

![image](https://cloud.githubusercontent.com/assets/203326/20004514/10698502-a28d-11e6-8934-b29d52822993.png)

![image](https://cloud.githubusercontent.com/assets/203326/20004533/3b44430c-a28d-11e6-873d-d4207e1bfbb4.png)

**Comment by Kewlname** [comment]
You are a legend for helping as much as you did. Much Love.

**Comment by peterix** [comment]
Hmmm... OK :)
